api/main_endpoints/routes/OfficeAccessCard.js

```diff
@@ -215,6 +216,57 @@ router.post('/getAllCards', async (req, res) => {
   }
 });
 
+router.post('/edit', async (req, res) => {
+  const decoded = decodeToken(req);
+  if (!decoded) {
+    return res.sendStatus(UNAUTHORIZED);
+  }
```

Comment:
i think we realized this doesnt work anymore, need to merge #1877 first

ill work on it